azure-devops/asan-pipeline.yml

```diff
@@ -4,36 +4,46 @@
 # Build STL targeting x86 and x64, and run extra ASan testing
 
 variables:
-  tmpDir: 'D:\Temp'
-  buildOutputLocation: 'D:\build'
-  # Restrict to "stlasan" test
-  testSelection: '-R stlasan'
-
-pool:
-  name: 'StlBuild-2023-09-14T1251-Pool'
-  demands: EnableSpotVM -equals true
+  - template: config.yml
+  - name: testSelection
+    value: '-R stlasan' # Restrict to "stlasan" test
 
 pr: none
+trigger: none
+
+schedules:
+- cron: '0 2 * * *'

```

Comment:
This starts a run every day at 02:00 UTC if there's been a change since the last scheduled run and no run is currently in progress (which should never matter: our runs never take 24 hours). I'm not married to 02:00, I chose it arbitrarily.